src/twisted/spread/pb.py

```diff
@@ -1425,13 +1633,24 @@ def remote_respond(self, response, mind):
         return d
 
 
-    # IUsernameHashedPassword:
     def checkPassword(self, password):
+        """
+        L{IUsernameHashedPassword}
+
+        @param password: The password.
+        @return: L{_PortalAuthChallenger.checkMD5Password}
```

Comment:
Pedantically, it doesn't return that.  It just returns a similar result given its similar input.